packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -1005,18 +984,30 @@ def test_ip_per_task_app_with_user_networking_and_portdefinitions(
             req_data = resp.json()
             assert req_data['endpoint_id'] == 'http://127.0.0.2:80'
 
-    def test_ip_per_task_app_without_user_networking(
-            self, nginx_class, mocker, valid_user_header):
+    @pytest.mark.parametrize('host_port', [12345, 0, None])
+    def test_app_with_container_networking_and_random_container_port(
+            self, nginx_class, mocker, valid_user_header, host_port):
+        # Testing the case when container port is specified as 0
+        # in Marathon app definition with networking mode 'container'.
+        # This means that the Marathon app container port is randomly assigned
+        # by Marathon. The chosen container port number is same as hostPort
+        # and is available through task["ports"] array. We are reusing port
```

Comment:
The container port is "randomly assigned" but also "is same as hostPort". Should the sentence "The chosen container port number is same as hostPort and is available through task["ports"] array" be in this comment?